src/packages/client/src/generation/TSClient.ts

```diff
@@ -281,9 +283,16 @@ export class TSClient implements Generatable {
       outputDir,
       schemaDir,
     } = this.options
+    const schemaPath = path.join(schemaDir, 'prisma.schema')
+    const envPaths = getEnvPaths(schemaPath, {cwd: outputDir})
+    const relativeEnvPaths = {
+      rootEnvPath: envPaths.rootEnvPath && path.relative(outputDir, envPaths.rootEnvPath),
+      schemaEnvPath: envPaths.schemaEnvPath && path.relative(outputDir, envPaths.schemaEnvPath)
+    }
 
-    const config: Omit<GetPrismaClientOptions, 'document' | 'dirname'> = {
+    const config: Omit<GetPrismaClientOptions, 'document' | 'dirname' > = {
```

Comment:
Let's remove the space again :D (unless prettier added it)
```suggestion
    const config: Omit<GetPrismaClientOptions, 'document' | 'dirname'> = {
```